frontend/src/components/common/progress-details/ProgressDetails.tsx

```diff
@@ -192,10 +192,22 @@ const ProgressDetails = ({
             <td className={'md'}>Sent to recipient</td>
             <td className={'sm'}>{sent}</td>
           </tr>
+          {type === ChannelType.Govsg && (
+            <tr>
+              <td className={cx(styles.status, 'md')}>
+                <i
+                  className={cx(styles.icon, styles.grey, 'bx bx-check-double')}
```

Comment:
Can we use `bx-mail-send` for delivered instead?

Reply:
I was thinking it looks like the grey double ticks on WhatsApp lol. but OK either way.

Reply:
Kendra has added to the slides alr — let's just merge as-is, if you don't mind terribly?

Reply:
Yea ok sorry let's go